src/CodeGen.fs

```diff
@@ -1203,4 +1214,114 @@ type {clientName}(url: string, httpClient: HttpClient) =
 let sampleFSharpGraphqlClient projectName clientName errorType members serializer =
     match serializer with
     | SerializerType.System -> sampleFSharpSystemGraphqlClient projectName clientName errorType members
-    | SerializerType.Newtonsoft -> sampleFSharpNewtonsoftGraphqlClient projectName clientName errorType members
\ No newline at end of file
+    | SerializerType.Newtonsoft -> sampleFSharpNewtonsoftGraphqlClient projectName clientName errorType members
+
+let sampleFableGraphqlClientFsi projectName clientName =
```

Comment:
This function is implemented incorrectly, Fable fsi implementation doesn't use any of `Newtonsoft.Json`, `Fable.Remoting.Json`, `System.Net.Http` and definitely no `HttpClient`

Reply:
@Zaid-Ajaj updated